nion/instrumentation/MultiAcquire.py

```diff
@@ -340,6 +343,7 @@ def cancel(self):
         self.abort_event.set()
         try:
             self.camera.acquire_sequence_cancel()
+            self.superscan.grab_synchronized_abort()
```

Comment:
This should call `acquire_sequence_cancel` automatically if `is_scanning` is `True`, which I think it would be here. Did you find otherwise?

Reply:
I think it does, but it also needs to work when we are not scanning, i.e. when MultiAcquire is used with individual spectra. Then it is just using `acuqire_sequence` so I think we need bot commands in here.
Or do you think this could cause any issues? If so I can keep an internal flag depending on which mode we are operating in and call the appropriate method.